curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-regular-expressions-by-building-a-spam-filter/66bb6467b4b7381178420970.md

```diff
@@ -0,0 +1,181 @@
+---
+id: 66bb6467b4b7381178420970
+title: Step 20
+challengeType: 0
+dashedName: step-20
+---
+
+# --description--
+
+While this expression does match `1 hundred dollars`, it will not however match `1  hundred  dollars`. Spam messages
```

Comment:
```suggestion
While this expression does match `1 hundred dollars`, it will not however match `1  hundred  dollars`, or `10 dollars`. Spam messages
```